Dockerfile.template

```diff
@@ -47,7 +47,8 @@ RUN set -ex; \
 		$PHPIZE_DEPS \
 		freetype-dev \
 		icu-dev \
-		imagemagick-dev \
+		imagemagick-dev libheif-dev \
+		libavif-dev \
```

Comment:
Ah, from the CI failure it looks like we need to add `libavif-dev` to the APT list below also (for the Debian-based variants).